src/pandroid/app/src/main/java/com/panda3ds/pandroid/utils/Constants.java

```diff
@@ -0,0 +1,26 @@
+package com.panda3ds.pandroid.utils;
+
+public class Constants {
+	public static final int INPUT_KEY_UP = 1 << 6;
+	public static final int INPUT_KEY_DOWN = 1 << 7;
+	public static final int INPUT_KEY_LEFT = 1 << 5;
+	public static final int INPUT_KEY_RIGHT = 1 << 4;
+
+	public static final int INPUT_KEY_A = 1 << 0;
+	public static final int INPUT_KEY_B = 1 << 1;
+	public static final int INPUT_KEY_X = 1 << 10;
+	public static final int INPUT_KEY_Y = 1 << 11;
+
+	public static final int INPUT_KEY_R = 1 << 8;
+	public static final int INPUT_KEY_L = 1 << 9;
+
+	public static final int INPUT_KEY_START = 1 << 3;
+	public static final int INPUT_KEY_SELECT = 1 << 2;
+
+	public static final int N3DS_WIDTH = 400;
+	public static final int N3DS_HALF_HEIGHT = 240;
```

Comment:
Probably better to define as `N3DS_FULL_HEIGHT / 2` to cut down on magic numbers